namnder/eventorganet/d-festeriet/body_en.md

```diff
@@ -0,0 +1,13 @@
+# D-festeriet
+
+
+
+D-Fest's tasks include - Organizing two parties for other officials per year, one after the Glögg-SM and one after the Val-SM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. - Organize regular team building for the chapter's officials. - Possibly organize other meetings and events together with other chapters and student unions to represent the chapter externally.
```

Comment:
This looks like a list that has lost it formatting when copying

```suggestion
D-Festeriet's tasks include: 

- Organizing two parties for other officials per year, one after the Glögg-SM and one after the Val-SM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. 

- Organize regular team building for the chapter's officials. 

- Possibly organize other meetings and events together with other chapters and student unions to represent the chapter externally.
```

Also in the swedish version there is some stuff about haltidsfesten that's missing in the English translation